src/ServiceControl.Persistence.RavenDB/CustomChecks/CheckMinimumStorageRequiredForIngestion.cs

```diff
@@ -77,11 +69,9 @@ public static void Validate(RavenPersisterSettings settings)
             }
         }
 
-        public const int MinimumStorageLeftRequiredForIngestionDefault = 5;
+        public const int MinimumStorageLeftRequiredForIngestionDefault = 20;
```

Comment:
As far as I understand 5 was never actually the default. Because when you don't set the threshold the default from the settings was applied which was 20

https://github.com/Particular/ServiceControl/pull/4026/files#diff-74de14602613e3840bb68469f19c4929908e0d830e211f05a9b30cb250a72361L426

It doesn't hurt to double check though as a reviewer because I got tired at some point staring at this code.

Reply:
It's documented as the default being 5: https://docs.particular.net/servicecontrol/creating-config-file

Looking at the two "disk check" custom checks, I think they need to have different default values, otherwise they don't seem to make a lot of sense having both.

It appears that the `DataSpaceRemainingThreshold` value for `CheckFreeDiskSpace` is intended to be the first alarm that disk space is getting low, and then the `MinimumStorageLeftRequiredForIngestion` value for `CheckMinimumStorageRequiredForIngestion` is the critical alarm, where ingestion is actually disabled.

Based on that, it seems like it make sense to keep/fix `MinimumStorageLeftRequiredForIngestion` at 5 and `DataSpaceRemainingThreshold` at 20.